packages/mobile/src/screens/update-required-screen/UpdateRequiredScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import { Platform, View } from 'react-native'
+
+import IconDownload from 'app/assets/images/iconDownload.svg'
+import { Button, GradientText, Text, useLink } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+export const ANDROID_PLAY_STORE_LINK =
+  'https://play.google.com/store/apps/details?id=co.audius.app'
+export const IOS_APP_STORE_LINK = 'itms-apps://us/app/audius-music/id1491270519'
+
+const messages = {
+  header: 'Update Required',
+  text: 'Behind minimum required version. Do the damn thing.',
+  buttonText: 'Update App'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  contentContainer: {
+    paddingTop: spacing(32),
+    paddingBottom: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  header: {
+    fontSize: 28,
```

Comment:
is there a fontSize that works in `typography.fontSize`? i guess if not, i wonder if we can either add one or maybe push back on design to use one of the established values?

Reply:
yea it might be good to add things for the headers